save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -302,8 +302,9 @@ class DockerService(private val configProperties: ConfigProperties) {
             .block()!!
     }?.map { it.toDto() } ?: emptyList()
 
-    @Suppress("UnsafeCallOnNullableType")
+    @Suppress("UnsafeCallOnNullableType", "TOO_MANY_LINES_IN_LAMBDA")
     private fun copyTestSuitesToResourcesPath(testSuitesForDocker: List<TestSuiteDto>, destination: File) {
+        destination.deleteRecursively()
```

Comment:
I suggest to use something like spring's `FileSystemUtils.deleteRecursively`, because it uses NIO under the hood

Reply:
Changed